### PR TITLE
New metric schema

### PR DESCRIPTION
**Pull request by tiffanyfj**

### CODE REVIEW

File: facter/facter.go [hunk 1]
```diff
@@ -140,11 +141,18 @@ func (f *Facter) GetMetricTypes(_ plugin.PluginConfigType) ([]plugin.PluginMetri
...
 	// parse and check requested names of metrics
 	names := []string{}
+	tags := make(map[string]string)
 	for _, metricType := range metricTypes {
+		tags = metricType.Tags()
+		if tags == nil {
+			tags = map[string]string{}
+		}
+		host, _ := os.Hostname()
+		tags["hostname"] = host
```
Comment (candysmurf) [inline facter/facter.go]: Just want to make sure that you like to add the hostname for each metric type, not the metric value.

Comment (tiffanyfj) [inline facter/facter.go]: Oops forgot to delete this. Thanks

**Comment by nanliu** [comment]
+1

**Comment by candysmurf** [comment]
need to update the Godeps.json to the latest Snap.

**Comment by tiffanyfj** [comment]
Godeps is at the latest version of snap
```
~/.gvm/pkgsets/system/global/src/github.com/intelsdi-x/snap git:master > git pull --rebase upstream master --tags                                                                                                                                             ✭
From github.com:intelsdi-x/snap
 * branch            master     -> FETCH_HEAD
Current branch master is up to date.
~/.gvm/pkgsets/system/global/src/github.com/intelsdi-x/snap git:master > ./build/bin/snapd -version                                                                                                                                                           ✭
snapd version v0.13.0-beta-77-g5c1177e
```

**Comment by candysmurf** [comment]
yes, it's. My bas. LGTM

**Pull request merged**